libsol/transaction_printers.c

```diff
@@ -552,6 +554,8 @@ static int print_transaction_nonce_processed(const PrintConfig* print_config,
                         print_config);
                 case ProgramIdSerumAssertOwner:
                 case ProgramIdSplMemo:
+                // If pointers are advanced before, this case won't be needed
```

Comment:
what does this comment mean?

Reply:
This is related to the comment below. `ProgramIdComputeBudget` is printed in slightly different way, so we don't want to return any print function here